src/infos.rs

```diff
@@ -10,3 +30,71 @@ static API_LEVEL: u8 = if cfg!(target_os = "nanos") {
 } else {
     0xff
 };
+
+#[used]
+#[link_section = "ledger.api_level"]
+static ELF_API_LEVEL: [u8;4] = if cfg!(target_os = "nanos"){
+    *b"0\n\0\0"
+} else if cfg!(target_os = "nanox") {
+    *b"5\n\0\0"
+} else if cfg!(target_os = "nanosplus") {
+    *b"1\n\0\0"
+} else {
+    *b"255\n"
+};
```

Comment:
Would need to be retrieved from the SDK, but this for after the change of architecture I guess?
Also, to be consistent with C sdk, the value for nanos should be : undefined, e.g. section doesn't exist.
and the else value might be the same ? Or a build error cause that's just invalid?

Reply:
Yes in the new SDK we will grab it from the C SDK, it is already done actually here https://github.com/LedgerHQ/secure-sdk-rust/blob/f78cb5803b47de24ef7b2b0323b8b08ad31f7399/build.rs#L180

Reply:
Done in https://github.com/LedgerHQ/ledger-device-rust-sdk/pull/96